relay-spans/src/span.rs

```diff
@@ -284,8 +284,8 @@ mod tests {
             "parentSpanId": "0c7a7dea069bf5a6",
             "name": "middleware - fastify -> @fastify/multipart",
             "kind": 1,
-            "startTimeUnixNano": 1697620454980000000,
-            "endTimeUnixNano": 1697620454980078800,
+            "startTimeUnixNano": "1697620454980000000",
+            "endTimeUnixNano": "1697620454980078800",
```

Comment:
Shouldn't we still be able to _parse_ long integers? Even if they get serialized as strings?

Reply:
Can you elaborate? `Even if they get serialized as strings?` -> They are serialized as a string and that works?

Reply:
Discussed offline: This will break the OTLP endpoint for old clients that send these values as numbers. We're OK with this breakage because the endpoint is experimental and gated behind a feature flag.